internal/state/game.go

```diff
@@ -0,0 +1,10 @@
+package state
+
+type Game struct {
```

Comment:
Should the game also have some kind of boundaries for where the entities can be located? I figure that we should just display a static board size, so knowing what that size is will be useful.

Reply:
yep, will plan to add boundaries in the next PR